### ISSUE TITLE
Show Veg/Non-Veg Indicator on the Food Items

### ISSUE THREAD

**Issue by ankit-ask** [Anandsg/Hungry-hero#294]
### Is your feature request related to a problem? Please describe.

To give the user a clear info about the differentiation between Veg/Non-Veg items, add relevant icons.

<img width="1246" alt="Screenshot 2023-10-31 at 18 32 47" src="https://github.com/Anandsg/Hungry-hero/assets/18434128/88859076-b563-4fee-af2b-c95cb217a129">

### Describe the solution you'd like

Add industry-standard GREEN or RED icons on top of Item names

<img width="826" alt="Screenshot 2023-10-31 at 18 34 46" src="https://github.com/Anandsg/Hungry-hero/assets/18434128/04094f87-b76a-4856-b940-2e3be38b1f09">

### Additional context

Please allow me to add this enhancement!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Anandsg** [comment]
yeah, this feature looks cool on this application please go ahead
